build.sbt

```diff
@@ -147,13 +162,13 @@ lazy val dependencies =
     val kafkaClient               = "org.apache.kafka"   % "kafka-clients"                          % "1.0.0"
     val zookeeper                 = "org.apache.zookeeper" % "zookeeper"                            % "3.4.9"
 
-    val json4s                    = "org.json4s"        %% "json4s-scalap"                          % json4sVersion
-    val jackson                   = "org.json4s"        %% "json4s-jackson"                         % json4sVersion
-    val json4sExt                 = "org.json4s"        %% "json4s-ext"                             % json4sVersion
+    val json4s                    = "org.json4s"          %% "json4s-scalap"                          % json4sVersion
+    val jackson                   = "org.json4s"          %% "json4s-jackson"                         % json4sVersion
+    val json4sExt                 = "org.json4s"          %% "json4s-ext"                             % json4sVersion
```

Comment:
Why is this weirdly formatted compared to the other dependencies?

Reply:
This was already the case but I fixed it for you